Kernel/Devices/HID/VirtIO/Input.cpp

```diff
@@ -111,7 +111,7 @@ static constexpr auto unshifted_evdev_key_map = to_array<KeyCodeEntry const>({
     { Key_Invalid, 0xff },           { Key_Mute, 0xff },              { Key_VolumeDown, 0xff },        { Key_VolumeUp, 0xff },
     { Key_Power, 0xff },             { Key_Equal, 0xff },             { Key_Invalid, 0xff },           { Key_PauseBreak, 0xff },
     { Key_Invalid, 0xff },           { Key_Invalid, 0xff },           { Key_Invalid, 0xff },           { Key_Invalid, 0xff },
-    { Key_Invalid, 0xff },           { Key_Super, 0xff },             { Key_Super, 0xff },             { Key_Menu, 0xff },
+    { Key_Invalid, 0xff },           { Key_LeftSuper, 0xff },         { Key_LeftSuper, 0xff },         { Key_Menu, 0xff },
```

Comment:
The right super key should be the second one
https://github.com/SerenityOS/serenity/blob/master/Kernel/Devices/HID/VirtIO/EvDevDefinitions.h#L215

Reply:
Thanks! This commit intends to just do a behavior-preserving rename. I thought I'd do behavior changes in a follow-up. (I also want to make things that check for left alt or left control check for both alts or both controls.)

Reply:
(Note to self: When changing this, I'll likely also have to update some key handling code to check for both right and left super then.)

Reply:
(If you want to test this you should set the `SERENITY_EXTRA_QEMU_ARGS` env var to `'-device virtio-tablet -device virtio-keyboard -M i8042=off'`)